ftw/mail/utils.py

```diff
@@ -37,6 +38,16 @@ def safe_decode_header(value):
     if apple_partial_encoding:
         value = ''.join(apple_partial_encoding.groups())
 
+    # Fix up RFC 2047 encoded words separated by 'CRLF LWSP' (which is fine
+    # according to the RFC) by replacing the CRLF with a SPACE so
+    # decode_header can parse them correctly.
+    # This works around a bug in decode_header that has been fixed in 3.3.
+    # See http://bugs.python.org/issue4491 and its duplicate.
+    # Example:
+    # From: =?utf-8?Q?B=C3=A4rengraben?=\r\n <from@example.org>
+    if ENCODED_WORD_WITHOUT_LWSP.match(value):
+        value = re.sub(ENCODED_WORD_WITHOUT_LWSP, '\\1 \\3', value)
```

Comment:
I think you could replace now with `'\\1\\3'`, since the 3rd group is a whitespace anyway and there is no need for an extra inserted space, right?